### PR TITLE
Empty system profiles XMLRPC tests

### PR DESCRIPTION
**Pull request by hustodemon**
## What does this PR change?
 
 Add cucumber tests for creating/listing empty system profiles.
 
 
 ## GUI diff
 
 No difference.
 
 - [x] **DONE**
 
 ## Documentation
 - No documentation needed: just tests
 
 - [x] **DONE**
 
 ## Test coverage
 - Only cucumber tests were added
 
 - [x] **DONE**
 
 ## Links
 
 Tracks https://github.com/SUSE/spacewalk/issues/5965
 
 - [x] **DONE**

### CODE REVIEW

**Comment by hustodemon** [comment]
gitarro no changelog needed !!!

**Comment by hustodemon** [comment]
Done.

**Comment by Bischoff** [comment]
OK, thanks, just added a small question / nitpick.

File: testsuite/features/min_empty_system_profiles.feature [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  Scenario: Create an empty minion profile with hostname via XML-RPC API
+    Given I am logged in via XML-RPC system as user "admin" and password "admin"
+    When I call system.create_system_profile() with name "empty-profile-hostname" and hostname "min-retail.mgr.suse.de"
+    And I logout from XML-RPC system namespace
+
+  Scenario: Check the created empty minion profiles in Unprovisioned Systems page
+    Given I am authorized
+    And I navigate to "rhn/systems/BootstrapSystemList.do" page
+    And I wait until I see "empty-profile" text, refreshing the page
+    And I wait until I see "00:11:22:33:44:55" text, refreshing the page
```
[the remark this answers is not in the document]
Comment (hustodemon) [reply testsuite/features/min_empty_system_profiles.feature]: Hmm, actually not. Good point! I'll try to fix.
